blocks/http/include/gnuradio-4.0/http/HttpBlock.hpp

```diff
@@ -245,18 +244,7 @@ class HttpBlock : public gr::Block<HttpBlock<T>, BlockingIO<false>, HttpBlockDoc
     }
 
     [[nodiscard]] constexpr auto
-    processOne(T value) noexcept {
```

Comment:
I wonder if `void processBulk(PublishableSpan auto &output)` would be a bit better here? 
Could the output port become a message port, too? Then the helper thread could probably write the messages directly into the port's writer?